ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -735,6 +783,9 @@ literals:
   - STANDBY
 - battery_charge_percentage_sensor
 - brightness_percentage_command
+- speed_mode:
```

Comment:
field has since been added, please add to the states on the existing one as needed and remove this one from PR